### PR TITLE
Feature/diet page

### PR DESCRIPTION
**Pull request by lenzloesche**
as written in the pull request.

### CODE REVIEW

File: apiServices/fetchDiet/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const url = "https://api.api-ninjas.com/v1/nutrition?query=";
+const apiKey = process.env.NEXT_PUBLIC_EXERCISES_APIKEY;
+
```
Comment (actyralf) [inline apiServices/fetchDiet/index.js:3]: This way your API key is stored on the client and will be visible to everyone.
It's better to create your own API route and route the results of the nutrition API through that.
Then your API key is not public and you can control access to it via your own API route.

File: components/Navigation/index.js [hunk 2]
```diff
@@ -52,6 +55,15 @@ export default function Navigation({selected, children, userName}) {
               Calendar
             </NavigationLink>
           </li>
+          {" - "}
+          <li>
+            <NavigationLink
+              selected={selected === "diet" ? true : false}
```
Comment (actyralf) [inline components/Navigation/index.js:61]: `selected={selected==="diet"}` would be sufficient

File: components/Navigation/index.js [hunk 3]
```diff
@@ -31,7 +32,9 @@ export default function Navigation({selected, children, userName}) {
               selected={selected === "index" ? true : false}
```
Comment (actyralf) [inline components/Navigation/index.js:32]: `selected={selected === "index"}`would be sufficient.

**Review by actyralf — CHANGES_REQUESTED** [review]
Well done!
One critical thing though: If you fetch the API directly from the client, you will expose your API key to the public.

File: apiServices/fetchDiet/index.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+const url = "https://api.api-ninjas.com/v1/nutrition?query=";
+const apiKey = process.env.NEXT_PUBLIC_EXERCISES_APIKEY;
+//const apiKey = NEXT_PUBLIC_EXERCISES_APIKEY;
```
Comment (manukao) [inline apiServices/fetchDiet/index.js]: should remove unused code before merging

File: apiServices/fetchDiet/index.js (hunk 1 above)
Comment (lenzloesche) [reply apiServices/fetchDiet/index.js:2]: thanks! this is great help and a big blunder.

**Review by manukao — APPROVED** [review]
nice, glad you were able to fix it the diet page.

**Pull request merged**